desktop/src/main/java/bisq/desktop/main/content/user/user_profile/UserProfileView.java

```diff
@@ -165,6 +174,19 @@ protected void onViewAttached() {
                 userIdentity -> comboBox.getSelectionModel().select(userIdentity));
     }
 
+    private void onSaveButtonPressed() {
+        if (runOnSaveValidations()) {
+            controller.onSave();
+        }
+    }
+
+    private boolean runOnSaveValidations() {
```

Comment:
Maybe isValid() would be a better method name?

Reply:
These are validations that are run only when saving (statement and terms length aren't validated on pressing on delete).